service/extensions/extensions.go

```diff
@@ -77,7 +82,8 @@ func (bes *Extensions) NotifyPipelineReady() error {
 func (bes *Extensions) NotifyPipelineNotReady() error {
 	// Notify extensions in reverse order.
```

Comment:
is this accurate?

Reply:
Looks like it's not. @atoulme can you please address this?

Reply:
How would you like me to address this? It feels like this comment needs to be removed.